src/popup/sass/layout/_search.scss

```diff
@@ -21,18 +21,21 @@
 
   i {
     position: absolute;
-    right: 0;
+    padding: 0;
+    margin: 0;
     top: 50%;
     transform: translateY(-50%);
     color: $color-secondary-light;
-
+    &#search-margin {
+      right : 4%;
+    }
```

Comment:
@Maxslide This was not `#search-margin` but `#search-icon`. This was also mentioned in the comment above.
Nevermind, it has been fixed in https://github.com/creativecommons/ccsearch-browser-extension/commit/5ac81e6f402cfe81b79bb45e65b93c45287e56a2